### DISCUSSION TITLE
Getting index.iife.min.js.map  as argument in server actions

### DISCUSSION THREAD

**Discussion by fitzmode** [vercel/next.js#67102] [Help]
### Summary

Thought to ask a question here because not quite able to replicate the issue. I have a server action in a server page.tsx component.

I keep getting `index.iife.min.js.map`  as an argument instead of  the `id` on the dynamic route /posts/[id]

const SinglePost = async ({ params }) => {
  const { id } = params;
  const position = await fetchPositionAction(id);
  
  return <div/>
  
  }
  
  In the `fetchPositionAction`, the id logs as `index.iife.min.js.map`.

**Comment by jim4067** [comment]
found this on stack overflow. Try closing the developer console. 

https://stackoverflow.com/a/78486456/11090641
